src/config/config.ts

```diff
@@ -0,0 +1,14 @@
+const config:any = (window as any).frontendConfig;
+// Override frontendConfig with localStorage frontendConfig if available
+if (localStorage.frontendConfig) {
+    try {
+        let localStorageFrontendConfig:any = JSON.parse(localStorage.frontendConfig);
+        for (var key in localStorageFrontendConfig) {
```

Comment:
not important but you could use Object.assign to do this merge.

Reply:
in fact, I would avoid using for in loop on objects because it will pick up properties from objects prototype.  unlikely, but has definately caused bug before.